SU-SOE/stanford_soe_helper
# 7.x 2.x jgk tweaks

# READY FOR REVIEW

# Summary
- Final style tweaks before launch/end of QA sprint

# Needed By (Date)
- 8/31 (end of sprint)

# Urgency
- N/A

# Steps to Test

1. Pull this branch
2. Verify new SCSS/CSS is being pulled - this branch has been pulled to /dev

# Affected Projects or Products
- SOE
- stanford_soe_helper

# Associated Issues and/or People
- SO many Jira tickets... :)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
